pennylane/ops/qubit/arbitrary_ops.py

```diff
@@ -0,0 +1,610 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
Reading first, arbitrary would mean that any operation could make it in here. Wondering: could `general_ops.py` work? This will likely be subjective, feel free to disagree.